README.md

```diff
@@ -225,6 +225,7 @@ vars:
   ansistrano_git_refspec: ADDITIONAL_GIT_REFSPEC # Additional refspec to be used by the 'git' module. Uses the same syntax as the 'git fetch' command.
   ansistrano_git_ssh_opts: "-o StrictHostKeyChecking=no" # Additional ssh options to be used in Git
   ansistrano_git_depth: 1 # Additional history truncated to the specified number or revisions
+  ansistrano_git_executable: "" # Path to git executable to use. If not supplied, the normal mechanism for resolving binary paths will be used.
```

Comment:
Maybe add a path like /opt/git or sth so that people see it needs to be a full path